cmd/vochaininspector/inspector.go

```diff
@@ -193,64 +187,65 @@ func newVochain(chain, dataDir string) *vochain.BaseApplication {
+// 		return nil, fmt.Errorf("wrong nullifier size %d", len(nullifier))
+// 	}
+// 	vid := sha256.New()
+// 	vid.Write(pid)
+// 	vid.Write(nullifier)
+// 	return vid.Sum(nil), nil
+// }
 
 func listBlockVotes(height int64, blockStoreDir string) {
-	cfg := tmcfg.DefaultConfig()
-	cfg.RootDir = blockStoreDir
-	blockStoreDB, err := node.DefaultDBProvider(&node.DBContext{ID: "blockstore", Config: cfg})
-	if err != nil {
-		log.Fatal("Can't open blockstore")
-	}
-	blockStore := store.NewBlockStore(blockStoreDB)
-	if blockStore == nil {
-		log.Fatal("Blockstore is nil")
-	}
-	block := blockStore.LoadBlock(height)
-	if block == nil {
-		log.Fatal("Block is nil")
-	}
-	fmt.Printf("Block txs: %v\n", len(block.Data.Txs))
-	for i, blockTx := range block.Data.Txs {
-		tx := new(vochain.VochainTx)
-		if err := tx.Unmarshal(blockTx, ""); err != nil {
-			log.Error(err)
-			continue
-		}
+	log.Fatal("listBlockVotes is not yet implemented since tendermint v0.35")
+	// cfg := tmcfg.DefaultConfig()
+	// cfg.RootDir = blockStoreDir
+	// blockStoreDB, err := node.DefaultDBProvider(&node.DBContext{ID: "blockstore", Config: cfg})
+	// if err != nil {
+	// 	log.Fatal("Can't open blockstore")
+	// }
+	// blockStore := store.NewBlockStore(blockStoreDB)
+	// if blockStore == nil {
+	// 	log.Fatal("Blockstore is nil")
+	// }
+	// block := blockStore.LoadBlock(height)
+	// if block == nil {
+	// 	log.Fatal("Block is nil")
+	// }
+	// fmt.Printf("Block txs: %v\n", len(block.Data.Txs))
+	// for i, blockTx := range block.Data.Txs {
+	// 	tx := new(vochain.VochainTx)
+	// 	if err := tx.Unmarshal(blockTx, ""); err != nil {
+	// 		log.Error(err)
+	// 		continue
+	// 	}
 
-		//	fmt.Printf("%+v\n", tx)
-		vote := tx.Tx.GetVote()
-		if vote == nil {
-			continue
-		}
-		pubKey, err := ethereum.PubKeyFromSignature(tx.SignedBody, tx.Signature)
-		if err != nil {
-			log.Fatalf("cannot extract public key from signature: %v", err)
-		}
-		addr, err := ethereum.AddrFromPublicKey(pubKey)
-		if err != nil {
-			log.Fatalf("cannot extract address from public key: %v", err)
-		}
-		vote.Nullifier = vochain.GenerateNullifier(addr, vote.ProcessId)
-		// fmt.Printf("%+v\n", vote)
-		vid, err := voteID(vote.ProcessId, vote.Nullifier)
-		if err != nil {
-			log.Fatalf("cannot get voteID: %v", err)
-		}
+	// 	//	fmt.Printf("%+v\n", tx)
+	// 	vote := tx.Tx.GetVote()
+	// 	if vote == nil {
+	// 		continue
+	// 	}
+	// 	pubKey, err := ethereum.PubKeyFromSignature(tx.SignedBody, tx.Signature)
+	// 	if err != nil {
+	// 		log.Fatalf("cannot extract public key from signature: %v", err)
+	// 	}
+	// 	addr, err := ethereum.AddrFromPublicKey(pubKey)
+	// 	if err != nil {
+	// 		log.Fatalf("cannot extract address from public key: %v", err)
+	// 	}
+	// 	vote.Nullifier = vochain.GenerateNullifier(addr, vote.ProcessId)
+	// 	// fmt.Printf("%+v\n", vote)
+	// 	vid, err := voteID(vote.ProcessId, vote.Nullifier)
+	// 	if err != nil {
+	// 		log.Fatalf("cannot get voteID: %v", err)
+	// 	}
```

Comment:
delete the code instead? leaving it commented out doesn't help - it's always in the git history if we need it.

Reply:
good catch: actually a no-op cli flag was being left behind, cleaned everything up in 1fadf6c